docs/bin/dump_keywords.py

```diff
@@ -77,7 +80,6 @@
 keyword_page = template.render(tempvars)
 if LooseVersion(jinja2.__version__) < LooseVersion('2.10'):
     # jinja2 < 2.10's indent filter indents blank lines.  Cleanup
-    keyword_page = re.sub(' +\n', '\n', keyword_page)
+    keyword_page = to_bytes(re.sub(' +\n', '\n', keyword_page))
```

Comment:
This isn't complete: for Python 3 (with a new enough Jinja2), it still breaks as `to_bytes` isn't called. Can you move the `to_bytes` to the `update_file_if_different()` call, like this?
```
update_file_if_different(outputname, to_bytes(keyword_page))
```